### PR TITLE
Fancy Scrollbars

### PR DESCRIPTION
**Pull request by alexpaxton**
- [x] CHANGELOG.md updated
- [x] Rebased/Mergeable
- [x] Tests Pass

### Problem
Was previously styling scrollbars using a `-webkit-` prefix. which only works in Chrome. In Firefox the scrollbars show up the browser default white/grey, which looks disturbingly bad against our dark UI

### Solution
Found a pretty sweet package called `react-custom-scrollbars` which is actively maintained, works in all browsers, and has surprisingly good documentation for being a small project. Very easy to customize. Now all users can experience aesthetically pleasing scrollbars despite their choice of browser, and Firefox users aren’t seeing ugly scrollbars.

### Preview
kap-rule-scrollbar
dropdown-scrollbar
scrollbar-fade

### CODE REVIEW

**Comment by alexpaxton** [comment]
Getting some weird issue in Safari in which no page renders, just the navbar. Currently investigating

**Comment by alexpaxton** [comment]
Figured out what was wrong, Safari was calculating the heights of the scroll containers as zero because their positions were set to `relative`, switched to `absolute` and everything worked okay. Damn you Safari

File: ui/src/style/components/tables.scss [hunk 1]
```diff
@@ -85,7 +85,7 @@ table .monotype {
     background-color: $c-pool;
   }
   &.dot-warning {
-    background-color: $c-comet;
+    background-color: $c-pineapple;
```
Comment (jaredscheib) [inline ui/src/style/components/tables.scss]: yessss

File: ui/src/kapacitor/components/KapacitorRule.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import RuleMessage from 'src/kapacitor/components/RuleMessage'
 import {createRule, editRule} from 'src/kapacitor/apis'
 import buildInfluxQLQuery from 'utils/influxql'
 import timeRanges from 'hson!../../shared/data/timeRanges.hson'
+import FancyScrollbox from 'src/shared/components/FancyScrollbar'
```
Comment (jaredscheib) [inline ui/src/kapacitor/components/KapacitorRule.js]: Let's call it the same as its file not Box vs Bar.

Also, it should be imported next to components, a few lines up, in every file.

File: ui/src/hosts/containers/HostPage.js [hunk 3]
```diff
@@ -7,6 +7,8 @@ import classnames from 'classnames'
 
 import LayoutRenderer from 'shared/components/LayoutRenderer'
 import DashboardHeader from 'src/dashboards/components/DashboardHeader'
+import FancyScrollbar from 'shared/components/FancyScrollbar'
+
```
Comment (jaredscheib) [inline ui/src/hosts/containers/HostPage.js]: nice

File: ui/src/hosts/containers/HostsPage.js [hunk 4]
```diff
@@ -1,9 +1,11 @@
 import React, {PropTypes} from 'react'
 import _ from 'lodash'
-import HostsTable from '../components/HostsTable'
-import SourceIndicator from '../../shared/components/SourceIndicator'
+
+import HostsTable from 'src/hosts/components/HostsTable'
```
Comment (jaredscheib) [inline ui/src/hosts/containers/HostsPage.js]: good call

File: ui/src/shared/components/FancyScrollbar.js [hunk 5]
```diff
@@ -33,10 +33,10 @@ class FancyScrollbox extends Component {
...
-FancyScrollbox.propTypes = {
+FancyScrollbar.propTypes = {
   children: node.isRequired,
   className: string,
   autoHide: bool,
 }
 
-export default FancyScrollbox
\ No newline at end of file
+export default FancyScrollbar
```
Comment (jaredscheib) [inline ui/src/shared/components/FancyScrollbar.js]: looks like we might want a carriage return at the end of this -- is webpack yelling at you for this?

File: ui/src/shared/components/TimeRangeDropdown.js [hunk 6]
```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import classnames from 'classnames'
-import OnClickOutside from 'shared/components/OnClickOutside'
-import FancyScrollbox from 'src/shared/components/FancyScrollbar'
-
 import moment from 'moment'
 
+import OnClickOutside from 'src/shared/components/OnClickOutside'
```
Comment (jaredscheib) [inline ui/src/shared/components/TimeRangeDropdown.js]: `src/` got added?

**Comment by jaredscheib** [comment]
LGTM

**Pull request merged**